manifests/poc-data-cluster/prod/spark/iot-rewards-silver.yaml

```diff
@@ -0,0 +1,81 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: iot-poc-silver-query
+  namespace: spark
+data:
+  query.sql: |
+    SELECT 
+      start_period,
+      end_period,
+      gateway_reward
+      FROM iot_reward_share
+---
+apiVersion: "sparkoperator.k8s.io/v1beta2"
+kind: SparkApplication
+metadata:
+  name: iot-poc-silver
+  namespace: spark
+spec:
+  type: Scala
+  mode: cluster
+  image: "public.ecr.aws/k0m1p4t7/spark:v3.4.0-aws"
+  imagePullPolicy: Always
+  mainClass: Main
+  mainApplicationFile: "s3a://foundation-data-lake-requester-pays/jars/spark-streaming-sql-assembly-1.0.jar"
+  sparkVersion: "3.4.0"
+  restartPolicy:
+    type: Never
+  sparkConf:
+    spark.databricks.delta.autoCompact.enabled: "true"
+  hadoopConf:
+    fs.s3a.aws.credentials.provider: com.amazonaws.auth.WebIdentityTokenCredentialsProvider
+  volumes:
+    - name: "tmp"
+      hostPath:
+        path: "/tmp"
+        type: Directory
+    - name: config-vol
+      configMap:
+        name: iot-poc-silver-query
+        items:
+          - key: query.sql
+            path: query.sql
+  driver:
+    serviceAccount: spark-data-lake-access
+    cores: 1
+    coreLimit: "1200m"
+    memory: "512m"
+    nodeSelector:
+      node.kubernetes.io/instance-type: m5.large
+    envVars:
+      TABLE_LORA_POC_V1: s3a://foundation-data-lake-requester-pays/bronze/lora_poc_v1
+      PARTITION_BY: "date"
+      CHECKPOINT: s3a://foundation-data-lake-requester-pays/checkpoints/iot-poc-silver
+      OUTPUT: s3a://foundation-data-lake-requester-pays/silver/iot-poc-witnesses
```

Comment:
fix output

Reply:
Renamed to TABLE_IOT_PACKETS.
A simpler name is better.  Nobody know what "packet reports v1" means.